### PR TITLE
Clarify that "kafka" is not a real storage backend

### PR DESCRIPTION
**Pull request by yurishkuro**
To address user questions like:

> Hi , I am exploring Jaeger with Kafka as an intermediate storage. I am unable to understand that why do we have Kafka producer as config for Jaeger query component. I believe that Jaeger query would always query the data from either Elastic or Cassandra? Can someone pls clarify? Thanks

### CODE REVIEW

File: cmd/env/command.go [hunk 1]
```diff
@@ -36,15 +36,31 @@ command line option                 environment variable
 
 The following configuration options are only available via environment variables:
 %s
+`
+	storageTypeDescription = `The type of backend [%s] used for trace storage.
+Multiple backends can be specified as comma-separated list, e.g. "cassandra,elasticsearch"
+(currently only for writing spans). Note that "kafka" is not a real storage backend, it is
```
Comment (vprithvi) [inline cmd/env/command.go]: I'd rephrase this to say "kafka" is distributed queue and not a complete storage solution and is expected to be used in conjunction with ingester.

Comment (yurishkuro) [reply cmd/env/command.go]: rephrased
